hubs.yaml

```
@@ -703,11 +703,10 @@ clusters:
             hub:
               config:
                 Authenticator:
-                  admin_users:
-                    - yuvipanda@2i2c.org
+                  admin_users: &mills_admins
                     - yuvipanda@gmail.com
-                    - choldgraf@2i2c.org
-                    - georgianaelena@2i2c.org
+                    - choldgraf@gmail.com
                     - georgiana.dolocan@gmail.com
                     - aculich@berkeley.edu
                     - jpercy@berkeley.edu
+                  username_pattern: '^.+@mills.edu$|yuvipanda@gmail.com|choldgraf@gmail.com|georgiana.dolocan@gmail.com|aculich@berkeley.edu|jpercy@berkeley.edu|deployment-service-check'
```

```suggestion
                  username_pattern: '^(.+@mills\.edu|yuvipanda@gmail\.com|choldgraf@gmail\.com|georgiana\.dolocan@gmail\.com|aculich@berkeley\.edu|jpercy@berkeley\.edu|deployment-service-check)$'
```

Regular expressions are an easy way to shoot yourself in the foot - particularly around '^' and '$'. Without those, the email 'notactuallyyuvipanda@gmail.com' would match the regex, since it'll match yuvipanda@gmail.com! Similar with '.', since it matches 'any character' regex.

Longer term we should do something else but this should do for now :D